pkg/sidecar/k8s_reactor.go

```diff
@@ -301,46 +302,6 @@ func (d *K8sReactor) manageContainer(ctx context.Context, container *docker.Cont
 		}
 	}
 
-	var addedGwRoute bool
```

Comment:
We have removed the code that deletes control network routes, as it has no impact on container performance in the new cluster. Instances (pods) communicate only over the data network.

Reply:
> They are used to set up the instances during startup, and to communicate with the sync service
Is there another use case for them?

This is another use case: https://github.com/testground/testground/pull/1481.

> I was a bit unclear when I said "Instances (pods) communicate only over the data network." - they communicate between themselves only over the data network.

Perfect, so I guess the EKS configuration takes on this responsibility now?
Could you link to the code that prevents instances from communicating with each other over the control network?

Reply:
Resolving, sorry for not closing this earlier, thanks for the clarification @brdji